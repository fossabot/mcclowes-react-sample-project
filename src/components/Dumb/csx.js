import styled from "styled-components";
import * as colors from "../../helpers/colors";

export const DumbWrapper = styled("div")``;
