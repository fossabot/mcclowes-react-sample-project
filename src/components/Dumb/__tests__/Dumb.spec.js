import Dumb from "../Dumb";
import React from "react";
import { shallow } from "enzyme";

describe("Dumb", () => {
  it("renders component", () => {
    const wrapper = shallow(<Dumb />);

    expect(wrapper).toMatchSnapshot();
  });
});
