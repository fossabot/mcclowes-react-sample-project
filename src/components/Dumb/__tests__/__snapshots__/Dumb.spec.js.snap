// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dumb renders component 1`] = `<styled.div />`;
